lib/shackles.rb

```diff
@@ -2,8 +2,11 @@
 
 module Shackles
   class << self
+    attr_accessor :primary_environment_name
+    Shackles.primary_environment_name = :primary
```

Comment:
Also, are we _sure_ defaulting to `:primary` here won't have downstream impacts? I found a few places in Switchman that appears to be making some assumptions about `:master` in the context of Shackles:
- https://github.com/instructure/switchman/blob/master/lib/switchman/shackles.rb#L11
- https://github.com/instructure/switchman/blob/80e951fccf3f63b0c3a4c08c837bb4a2b463b330/lib/switchman/connection_pool_proxy.rb#L34
- https://github.com/instructure/switchman/blob/c1ae86f21d9e75371b06dc6791b59eb3837450ee/lib/switchman/database_server.rb#L72

Reply:
yup. that's why we bump the version of shackles so that switchman (or any of its users) won't automatically pull in this new version. then in switchman, we update those at the same time that we allow a newer version of shackles.